packages/phone-number-privacy/combiner/src/pnp/services/thresholdState.ts

```diff
@@ -0,0 +1,21 @@
+import { PnpQuotaRequest, PnpQuotaResponseSuccess } from '@celo/phone-number-privacy-common'
+import { Session } from '../../common/session'
+import { OdisConfig } from '../../config'
+
+export class CombinerThresholdStateService<R extends PnpQuotaRequest> {
+  constructor(readonly config: OdisConfig) {}
+
+  findCombinerQuotaState(session: Session<R>) {
+    const signerResponses: PnpQuotaResponseSuccess[] = session.responses
+      .map((signerResponse) => signerResponse.res)
+      .filter((res) => res.success)
+      .sort((a, b) => a.totalQuota - b.totalQuota)
```

Comment:
```suggestion
      .sort((a, b) => (a.totalQuota - a.performedQueryCount) - (b.totalQuota - b.performedQueryCount)
```
The way this works and the naming is a bit confusing :/ feel free to suggest improvements!